telemetry/definitions/commonDefinitions.json

```
@@ -519,6 +519,18 @@
             "allowedValues": ["java", "python", "javascript", "plaintext", "jsx", "typescript", "tsx", "csharp",
                 "c", "cpp", "go", "kotlin", "php", "ruby", "rust", "scala", "shell", "sql"]
         },
+        {
+            "name": "codewhispererGettingStartedTaskType",
+            "type": "string",
+            "description": "The task type selected by the user in getting started page",
+            "allowedValues": [
+                "AUTO_TRIGGER",
+                "MANUAL_TRIGGER",
+                "COMMENT_AS_PROMPT",
```

why are these uppercase?
```suggestion
                "commentAsPrompt",
```

---

Updated.